router/src/main.rs

```diff
@@ -328,6 +328,15 @@ async fn main() -> Result<(), RouterError> {
     tracing::info!("Setting max batch total tokens to {max_supported_batch_total_tokens}");
     tracing::info!("Connected");
 
+    // Determine the server port based on the feature and environment variable.
+    let port = if cfg!(feature = "google") {
```

Comment:
Let's make that compile time.

```rust
#[cfg(feature = "google")]
let port = xxx

#[cfg(not(feature = "google"))]
let port = xxxx
```

Reply:
Nvm, I checked the macros, it's also compile time.

Reply:
nice, however if helpful I can update the syntax